server/routes/user.py

```diff
@@ -50,14 +53,15 @@ def login():
             },
             app.config["SECRET_KEY"],
         )
-        return jsonify({"token": token})
+    return jsonify({"token": token, "message": "Login Successful!"})
```

Comment:
Was this change intended? :thinking: Because if the code doesn't enter the `if` block, `token` won't be defined and the function will mostly like break and return as a 500.

Reply:
Ohh yes....I guess indentation error 
My bad 😅 
I'll make the required changes 🚀